lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/utils/MultiRuntimeDefinitionSerializer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.reef.runtime.multi.utils;
+
+import org.apache.avro.io.*;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.apache.avro.specific.SpecificDatumWriter;
+import org.apache.reef.runtime.multi.utils.avro.MultiRuntimeDefinition;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * Serializer for MultiRuntimeDefinition.
+ */
+public final class MultiRuntimeDefinitionSerializer {
+
+  private static final String CHARSET_NAME = "UTF-8";
+
+  /**
+   * Serializes MultiRuntimeDefinition.
+   * @param runtimeDefinition the Avro object to serialize
+   * @return Serialized avro string
+   */
+  public String serialize(final MultiRuntimeDefinition runtimeDefinition){
```

Comment:
in other cases, we have used `toString(...)` and `fromString(...)` for these method names. That allows us to add other serialization targets and removes ambiguity when reading the code at the call site.